packages/es-dev-server/src/utils/utils.js

```diff
@@ -114,6 +114,22 @@ export class SSEStream extends Stream.Transform {
   }
 }
 
+/**
+ * @param  {*}  x
+ * @return {x is boolean}
+ */
+export function isBoolean(x) {
+  return typeof x === 'boolean';
+}
+
+/**
+ * @param  {boolean|import('chokidar').WatchOptions}  watch
+ * @return {watch is WatchOptions}
+ */
+export function isWatchOptions(watch) {
```

Comment:
This isn't used anywhere?

Reply:
true. see previous comment. seems typescript preferred the opposite predicate 🤷‍♂

Reply:
removed